FraihmworkHealthAndIntegrityApi.yaml

```diff
@@ -696,6 +696,117 @@ paths:
       security:
       - OAuth2: [fraihmwork.component.write, fraihmwork.component.admin]
 
+  /monitor/v1/component/{uuid}/timeOfPerformance:
+    post:
+      tags:
+      - Components
+      - Monitor
+      summary: Resets the time of performance 
+      description: Given a specific component UUID, this operation establishes a new start time for the measurement of the 'time of performance' field tracked by components. When received, the server immediately sets the start time to the current clock time. Note, requests with the fraihmwork.component.write permission will only be accepted if the client used is the 'owner' of the component. Otherwise a 403 forbidden will be returned. See the descriptions in the securitySchemes.OAuth2.flows.clientCredentials.scopes for more details.     
+      operationId: resetTimeOfPerformance
+      parameters:
+        - in: path
+          name: uuid
+          required: true
+          schema:
+            type: string
+            format: uuid
+          description: Uuid of the component for Time of Performance to be reset
+      responses:
+        200:
+          description: Time of performance successfully updated
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+          
+        400:
+          description: Bad Request (invalid submission)
+          content:
+            application/json:
+              schema:
+                $ref: '#/components/schemas/Response'
+        401:
+          description: Unauthorized
```

Comment:
Comparing against the /refresh endpoint above, this description is not the same. Same for other 401 added. Sync up.

Reply:
Updated